config/label-management.md

```diff
@@ -0,0 +1,126 @@
+# Managing cross-organization labels in existing repositories #
+
+This is a step-by-step guide/script for managing organization-wide GitHub
+issue labels across all repositories.
+
+## Prerequisites ##
+
+- User has access to the GitHub organization's repositories
+- User can generate a GitHub [Personal access token](https://github.com/settings/tokens)
+
+## Initial Setup ##
+
+1. Clone the repository locally
+
+    ```console
+    git clone https://github.com/alta3/github-label-management.git
+    cd github-label-management
+    ```
+
+1. Activate/create the virtual environment and install the requirements
+
+    ```console
+    pyenv local 3.8.3
+    pyenv virtualenv github-label-management
+    pyenv activate github-label-management
```

Comment:
This should align with how we tell contributors to setup their Python environments. You can see [`cisagov/skeleton-generic`'s `CONTRIBUTING.md`](https://github.com/cisagov/skeleton-generic/blob/develop/CONTRIBUTING.md#creating-the-python-virtual-environment) for an example.

Does it need to be that specific version of Python? Python 3.8 is at `3.8.6` patch version now.
```suggestion
    pyenv virtualenv 3.8.3 github-label-management
    pyenv local github-label-management
```